RecoBTag/FeatureTools/interface/SeedingTracksConverter.h

```diff
@@ -0,0 +1,35 @@
+#ifndef RecoBTag_FeatureTools_SeedingTracksConverter_h
+#define RecoBTag_FeatureTools_SeedingTracksConverter_h
+
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+#include "DataFormats/ParticleFlowCandidate/interface/PFCandidate.h"
+
+#include "DataFormats/BTauReco/interface/SeedingTrackFeatures.h"
+#include "DataFormats/BTauReco/interface/TrackPairFeatures.h"
+
+#include "RecoBTag/FeatureTools/interface/TrackPairInfoBuilder.h"
+#include "RecoBTag/FeatureTools/interface/SeedingTrackInfoBuilder.h"
+
+#include "DataFormats/PatCandidates/interface/Jet.h"
+
+#include "TrackingTools/TransientTrack/interface/TransientTrackBuilder.h"
+#include "TrackingTools/IPTools/interface/IPTools.h"
+#include "TrackingTools/Records/interface/TransientTrackRecord.h"
+#include "RecoVertex/VertexTools/interface/VertexDistance3D.h"
+
+#include "RecoBTag/TrackProbability/interface/HistogramProbabilityEstimator.h"
+
+namespace btagbtvdeep {
+    
+        void seedingTracksToFeatures(edm::Handle<edm::View<reco::Candidate> > tracks,
+                                        const reco::Jet & jet,
+                                        const reco::Vertex & pv,                                            
+                                        edm::ESHandle<TransientTrackBuilder> & track_builder,
+                                        HistogramProbabilityEstimator* probabilityEstimator,
+                                        bool computeProbabilities,
+                                        std::vector<btagbtvdeep::SeedingTrackFeatures> & seedingT_features_vector
```

Comment:
from https://github.com/cms-sw/cmssw/pull/26088#discussion_r280585227
> given max size of 10 and zero padding, does it make sense to keep this a vector instead of an `std::array< ... , 10>` ?

Reply:
std:vector is used in analogy to deepFlavour jet features https://github.com/leonardogiannini/cmssw/blob/deepvertex105x/DataFormats/BTauReco/interface/DeepFlavourFeatures.h#L21-L24. Should I move to std::array anyway?

Reply:
ah, good point.
No, no need to change to an array in this case